README.md

```diff
@@ -39,11 +39,26 @@ http.cors.enabled: true
 http.cors.allow-origin: "*"
 ```
 
-### Index settings
+### OpenSearch details
 
 ![OpenSearch data source details](https://raw.githubusercontent.com/grafana/opensearch-datasource/main/docs/img/opensearch-details.png)
 
-Here you can specify a default for the `time field` and specify the name of your OpenSearch index. You can use
+- **Index name** - Use the index settings to specify a default for the `time field` and your Elasticsearch index's name. You can use a time pattern, for example `[logstash-]YYYY.MM.DD`, or a wildcard for the index name. When specifying a time pattern, the fixed part(s) of the pattern should be wrapped in square brackets.
```

Comment:
Elasticsearch -> OpenSearch.

Should "Use the index settings to specify a default for the `time field`" be this instead "specify the name of your OpenSearch index. You can use a time pattern, for example `[logstash-]YYYY.MM.DD`, or a wildcard for the index name."

I.e. getting rid of the referenc to `time field` here?